### DISCUSSION TITLE
Different UI than advertised?

### DISCUSSION THREAD

**Discussion by Tectrek** [bkaankose/Wino-Mail#336] [Q&A] [answered]
Heya!

I might be just stupid, but where is the option to enable to UI that can be seen in basically every picture of this App?
When i downloaded the program every eMail account was seperated from the rest, so i could never see the Inboxes of all acounts at the same time.
![image](https://github.com/user-attachments/assets/bb809ddf-7243-4857-9a3d-d614415b0f27)

While on all the pictures every account is useable at the same time and collapseable.
![image](https://github.com/user-attachments/assets/64a0592e-5ee3-4ffb-a69b-aea05ac43383)

Im personally a big fan of how Thunderbird handles that, where you can technically see all folders/accounts ,but can favorite those which are the most important for fast access to them. 
![image](https://github.com/user-attachments/assets/26ec9a36-1a03-4fe1-8e15-50aec7e5c33e)

The current UI is quite the exact opposite. Especially because collapseable Folders always get shown in their collapsed state, when i click on the corresponding Account, what is a pain in the rear. So i dont just have to do extra-clicks to even see what type of Mail i got on each account, but also have to make extra clicks to even see them in their folders.
So... if there is an option to get the (i guess) old UI back, then please let me know. I would realy like to ditch Thunderbird and their thin-ass fonts and crappy Dark Modes for the Reader.

Would also be nice if there were an option to put the program in the system tray and minimize it onto it when closing. I dont see the need to have the program taking up space in my task-bar just to see when i got a new Mail.

I hope my critique is understandable and wish everyone a nice day! :)

**Comment by bkaankose** [comment, marked as answer]
Hi,

Current UI is implemented to copy exact behavior of Windows Mail & Calendar. People find it more intuitive :) You are the only person so far that wants to use the old UI back, but unfortunately maintaining both of them is too much to handle at this point.

One thing that makes sense to have though is always pinned combined folders. People may want to combine multiple folders from different accounts and have them listed in the list always. This is something we don't have right now but I can add it to the backlog as an improvement.

Regarding the system tray mechanism, it is implemented with Beta versions, soon will be available in the Microsoft Store as an update with a lot of improvements. If you can find information about the beta releases in readme of the repository.

**Reply by Tectrek** [reply]
Then i would at least change the images in the Store and on the website. Its quite misleading how it currently is. :D 
Tho thats quite a bummer to be honest. No idea how this is supposed to be more intuitive, but i guess it is how it is. :/

Then i will keep an eye on this project, but probably wont use it. Still wish you the best going forward and i think the Tray-function is a good step forward. :)

Have a nice one! :D

**Reply by HumanError13** [reply]
> You are the only person so far that wants to use the old UI back

Please count two! ;)

I'm also used to this behaviour from Thunderbird and Outlook. I did not use Wino Mail yet, but really liked to see the mail accounts all at once on the screenshots. Now reading this, I probably won't use Wino Mail. This is a feature I really need, unfortunately. And I don't understand as well why people want to have their accounts viewed separately.

I also will keep an eye on this project, however. It's amazing what you did here.
